testing/test_modules.sh

```diff
@@ -10,14 +10,9 @@ cp config/system/all.conf local/system.conf
 TEST_LIST=/tmp/module_tests.txt
 
 cat > $TEST_LIST <<EOF
-ping
```

Comment:
why remove all these?

Reply:
test_modules only work with docker containers setups; I don't know if it's worth the investment to rewrite another one to work with 2 native tests.

Reply:
fair enough. As long as it's all being tested somewhere (which it is still in the integration tests) then we're all good. I agree that doing more work in this direction doesn't make sense until we have the wider team trying to do native test development.